src/kubernetes/server/server-deployment.yaml

```diff
@@ -18,6 +18,10 @@ spec:
       - name: scdf-server
         image: springcloud/spring-cloud-dataflow-server:2.5.0.BUILD-SNAPSHOT
         imagePullPolicy: Always
+        volumeMounts:
```

Comment:
these changes need to go into `src/templates/kubernetes/server/server-deployment.yaml` (note the `templates` in the path) otherwise the build process won't get them /cc @jvalkeal